src/coreclr/scripts/genEventPipe.py

```diff
@@ -593,7 +593,6 @@ def getMonoEventPipeHelperFileImplPrefix():
     size_t *size,
     bool *fixed_buffer)
 {
-    EP_ASSERT (value != NULL);
```

Comment:
We have case where we have struct of size 0 and then value will be NULL. So instead of doing some ugly solution in caller to make sure we always have a pointer, even when size is 0, we could just remove the assert since it is a valid case and existing code handles it, so no need to assert.